### ISSUE TITLE
Input setup from CSV file

### ISSUE THREAD

**Issue by Hyzbade** [samthiriot/genlab#57]
Currently, we are able to run Watts-Strogatz n times in a row with a specific setup of inputs to get some statistics. In other words, we can initialize our inputs then we launch the simulation.
Now, if we could set up these inputs from a CSV file which is containing many setup of inputs, it will be awesome!

**Comment by samthiriot** [comment]
Good idea ! Would require at first glance a "read table" algorithm, and a specific container that would take data from the table and would use it as inputs. Feasible and interesting, but not trivial to implement...
